packages/synapse-react-client/src/components/CreateOrUpdateAccessRequirementWizard/CreateOrUpdateAccessRequirementWizard.tsx

```diff
@@ -0,0 +1,159 @@
+
+  const isEditing = Boolean(props['accessRequirementId'])
+
+  const stepContent = useMemo(() => {
+    switch (step) {
+      case 'SET_AR_COMMON_FIELDS':
+        return (
+          <SetAccessRequirementCommonFields
+            ref={setArCommonFieldsRef}
+            onSave={(accessRequirementId, accessRequirementConcreteType) => {
+              setAccessRequirementId(accessRequirementId)
+              const nextStep: CreateOrUpdateAccessRequirementWizardStep =
+                accessRequirementConcreteType ===
+                MANAGED_ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE
+                  ? 'SET_MANAGED_AR_FIELDS'
+                  : 'SET_BASIC_AR_FIELDS'
+              setStep(nextStep)
+              setIsLoading(false)
+            }}
+            onError={() => setIsLoading(false)}
+            subject={props.subject}
+            accessRequirementId={props.accessRequirementId}
+          />
+        )
+      case 'SET_MANAGED_AR_FIELDS':
+        return (
+          <SetManagedAccessRequirementFields
+            ref={setManagedArFieldsRef}
+            accessRequirementId={accessRequirementId!}
+            onSave={() => {
+              setStep('SET_MANAGED_AR_ACL_PERMISSIONS')
+              setIsLoading(false)
+            }}
+            onError={() => {
+              setIsLoading(false)
+            }}
+          />
+        )
+      case 'SET_MANAGED_AR_ACL_PERMISSIONS':
+        return (
+          <AccessRequirementAclEditor
+            ref={editArAclRef}
+            accessRequirementId={accessRequirementId!}
+            onSaveComplete={saveSuccessful => {
+              if (saveSuccessful) {
+                onComplete()
+              }
+              setIsLoading(false)
+            }}
+          />
+        )
+      case 'SET_BASIC_AR_FIELDS':
+        return (
+          <SetBasicAccessRequirementFields
+            ref={setBasicArFieldsRef}
+            accessRequirementId={accessRequirementId!}
+            onSave={() => {
+              onComplete()
+              setIsLoading(false)
+            }}
+            onError={() => setIsLoading(false)}
+          />
+        )
+    }
+  }, [
+    step,
+    props.subject,
+    props.accessRequirementId,
+    accessRequirementId,
+    onComplete,
+  ])
+
+  const onClickPrimary = () => {
+    setIsLoading(true)
+    switch (step) {
+      case 'SET_AR_COMMON_FIELDS':
+        setArCommonFieldsRef?.current?.save()
+        return
+      case 'SET_MANAGED_AR_FIELDS':
+        setManagedArFieldsRef?.current?.save()
+        return
+      case 'SET_MANAGED_AR_ACL_PERMISSIONS':
+        editArAclRef?.current?.save()
+        return
+      case 'SET_BASIC_AR_FIELDS':
+        setBasicArFieldsRef?.current?.save()
+    }
+  }
+
+  return (
+    <ConfirmationDialog
+      open={open}
+      onCancel={onCancel}
+      onConfirm={onClickPrimary}
+      confirmButtonProps={{
+        children: isLastStep(step) ? 'Save' : 'Save & Continue',
```

Comment:
Changed primary button copy from "Next" to "Save & Continue" to reflect that each step saves the AR